charts/accurate/templates/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
This file should be generated with controller-gen.
Could you auto-generate the template from the output of controller-gen?